models/experimental/functional_bert/demo/demo.py

```diff
@@ -0,0 +1,312 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+import pytest
+import torch
+from loguru import logger
+
+import transformers
+import ttnn
+import tt_lib
+from models.utility_functions import (
+    disable_compilation_reports,
+    disable_persistent_kernel_cache,
+    profiler,
+)
+from models.experimental.functional_bert.tt import ttnn_functional_bert
+from models.experimental.functional_bert.tt import ttnn_optimized_functional_bert
+
+from models.datasets.dataset_squadv2 import squadv2_1K_samples_input, squadv2_answer_decode_batch
+from ttnn.model_preprocessing import (
+    preprocess_model_parameters,
+)
+
+from ttnn.model_preprocessing import *
+from transformers import BertForQuestionAnswering, BertTokenizer, pipeline
+
+import evaluate
+
+
+def load_inputs(input_path, batch):
+    with open(input_path) as f:
+        input_data = json.load(f)
+        assert len(input_data) >= batch, f"Input data needs to have at least {batch} (batch size) entries."
+
+        context = []
+        question = []
+        for i in range(batch):
+            context.append(input_data[i]["context"])
+            question.append(input_data[i]["question"])
+
+        return context, question
+
+
+def run_bert_question_and_answering_inference(
+    device,
+    use_program_cache,
+    model_name,
+    batch_size,
+    sequence_size,
+    functional_bert,
+    model_location_generator,
+    input_path,
+):
+    disable_persistent_kernel_cache()
+
+    model = str(model_location_generator(model_name, model_subdir="Bert"))
+    hugging_face_reference_model = BertForQuestionAnswering.from_pretrained(model, torchscript=False)
+    hugging_face_reference_model.eval()
+
+    # set up tokenizer
+    tokenizer_name = str(model_location_generator(model_name, model_subdir="Bert"))
+    tokenizer = BertTokenizer.from_pretrained(tokenizer_name)
+    config = hugging_face_reference_model.config
+    nlp = pipeline("question-answering", model=hugging_face_reference_model, tokenizer=tokenizer)
+
+    if functional_bert == ttnn_functional_bert:
+        tt_model_name = f"ttnn_{model_name}"
+    elif functional_bert == ttnn_optimized_functional_bert:
+        tt_model_name = f"ttnn_optimized_{model_name}"
```

Comment:
Please change this to `ttnn_{model_name}_optimized`

Reply:
`f"ttnn_optimized_{model_name}"` is changed to `ttnn_{model_name}_optimized`